sql/2024/fonts/design/fonts_designer.sql

```diff
@@ -0,0 +1,52 @@
+-- Section: Design
+-- Question: Which designers are popular?
+-- Normalization: Sites
+
+-- INCLUDE ../common.sql
+
+WITH
+designers AS (
+  SELECT
+    client,
+    NULLIF(TRIM(JSON_EXTRACT_SCALAR(payload, '$._font_details.names[9]')), '') AS designer,
+    COUNT(DISTINCT page) AS count,
+    ROW_NUMBER() OVER (PARTITION BY client ORDER BY COUNT(DISTINCT page) DESC) AS rank
+  FROM
+    `httparchive.all.requests`
+  WHERE
+    date = '2024-07-01' AND
```

Comment:
I thought we were using 2024-06-01 for web Almanac. Is there a reason we can’t use this?

Reply:
We changed the crawler after the 2024-06-01 data were collected.

Reply:
@nrllh FYI, just making sure you're cool with this?

Reply:
That's okay IMO